cli/utilities/operations.py

```diff
@@ -24,3 +27,30 @@ def wait_for_cluster_health(node, status, timeout=300, interval=20):
     if w.expired:
         log.error(f"Cluster is not in {status} state even after {timeout} secs")
         return False
+
+
+def wait_for_osd_daemon_state(client, id, state):
```

Comment:
Can this be a generic one? I mean wait_for_daemon_state(daemon, client, id, state) ?

Not an urgent change, but just a thought

Reply:
Different daemons has different commands to get status, which will make the function more complex, hence created separate method.